src/views/cart.hbs

```diff
@@ -0,0 +1,38 @@
+<main class="cart">
+    <section>
+        <div class="cart-header">
+            <div class="cart-header__text">My Cart {{cart.count}} items</div>
+        </div>
+        {{#each cart.items}}
+        <div class="cart-item" data-id="{{product.id}}">
+            <img src="{{product.imageURL}}" alt="{{product.name}}" class="cart-item__logo" />
+            <div class="cart-item-detail">
+                <h2 class="cart-item-detail__heading">{{product.name}}</h2>
+                <div class="cart-item-detail__info">
+                    <div class="cart-item-detail__info--count">
+                        <a class="cart-item-detail__info--remove" data-id="{{product.id}}">&#45;</a>
+                        <span class="cart-item-detail__info--item-count" data-id="{{product.id}}">{{count}}</span>
+                        <a class="cart-item-detail__info--add" data-id="{{product.id}}">&#43;</a>* Rs.{{product.price}}
```

Comment:
As you are making a website, how are you ensuring consistency in re-usable elements like anchor tags, paragraphs and headings